ctest.h

```diff
@@ -16,6 +16,10 @@
 #ifndef CTEST_H
 #define CTEST_H
 
+#if !(defined (_GNU_SOURCE) || defined (_BSD_SOURCE) || defined(_POSIX_C_SOURCE))
+#undef CTEST_SEGFAULT
```

Comment:
Not sure why this is necessary. While `kill` isn't standard, in the particular case below it could be substituted with `raise(signum);` which is standard alright. Am I missing something?